src/rendering/dashboarditem.cpp

```diff
@@ -37,10 +37,16 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo EnabledInfo = {
         "Enabled",
         "Enabled",
-        "If this value is set to 'true' this dashboard item is shown in the dashboard.",
-        openspace::properties::Property::Visibility::AdvancedUser
+        "If this value is set to 'true' this dashboard item is shown in the dashboard. "
+        "Otherwise it will be hidden."
     };
 
+    // A `DashboardItem` is an item that is rendered in a dashboard. There are multiple
+    // dashboards that can exist in the system; the default dashboard is rendered to the
```

Comment:
Maybe nitpicky, but not sure what I think of the use of the semicolon here. Might make the text harder to read than necessary if the reader is not familiar with it.

Reply:
Replaced it with:
"There are multiple dashboards that can exist in the system. One dashboard is created by default, which is rendered to the top left of the screen. Others can be added [...]"